src/NewsService.Mappers/Responses/NewsResponseMapper.cs

```diff
@@ -0,0 +1,104 @@
+﻿using LT.DigitalOffice.Broker.Requests;
+using LT.DigitalOffice.Broker.Responses;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Kernel.Exceptions.Models;
+using LT.DigitalOffice.NewsService.Mappers.ResponsesMappers.Interfaces;
+using LT.DigitalOffice.NewsService.Models.Db;
+using LT.DigitalOffice.NewsService.Models.Dto.Models;
+using LT.DigitalOffice.NewsService.Models.Dto.Responses;
+using MassTransit;
+using Microsoft.Extensions.Logging;
+using System;
+
+namespace LT.DigitalOffice.NewsService.Mappers.Responses
+{
+    public class NewsResponseMapper : INewsResponseMapper
+    {
+        private IRequestClient<IGetUserDataRequest> _userRequestClient;
+        private IRequestClient<IGetDepartmentRequest> _departmentRequestClient;
+        private readonly ILogger _logger;
+
+        private string GetUserFIO (Guid userId)
+        {
+            string fio = null;
+
+            try
+            {
+                var request = IGetUserDataRequest.CreateObj(userId);
+                var response = _userRequestClient.GetResponse<IOperationResult<IGetUserDataResponse>>(request).Result;
+
+                if (!response.Message.IsSuccess)
+                {
+                    _logger.LogWarning($"Can't found user FIO. Reason: '{string.Join(',', response.Message.Errors)}'");
+                }
+
+                fio = $"{response.Message.Body.LastName} {response.Message.Body.FirstName} {response.Message.Body.MiddleName}".Trim();
```

Comment:
FIO = Familya Imya Otchestvo? Rename to 'fullName'

Reply:
done